internal/config/specs.go

```diff
@@ -32,7 +32,7 @@ type EnvSpec struct {
 	OAuth2ClientSecret              string        `envconfig:"oauth2_client_secret" validate:"required"`
 	OAuth2RedirectURI               string        `envconfig:"oauth2_redirect_uri" validate:"required"`
 	OAuth2CodeGrantScopes           []string      `envconfig:"oauth2_codegrant_scopes" default:"openid,offline_access" validate:"required"`
-	OAuth2NonceCookieTTL            time.Duration `envconfig:"oauth2_nonce_cookie_ttl" default:"5m0s" validate:"required"`
+	OAuth2AuthCookiesTTL            time.Duration `envconfig:"oauth2_auth_cookies_ttl" default:"5m0s" validate:"required"`
```

Comment:
as env var this is a bit non standard in terms of value, i would say it's easier to just have it in seconds so it becomes a plain number (and hopefully we would be able to import it and parse)

i cant really see `OAUTH2_AUTH_COOKIES_TTL=30m` being easy to use, but happy to stick with it if parsing is to hard to implement right now

Reply:
Duration parsing is usually pretty standard for extrenalized configs, but I can change it OAuth2AuthCookieTTLSeconds and make it an int, no problem.
I'll push this change with the missing texts as soon as I have them